### DISCUSSION TITLE
Add 90 - 180° rotation: feature enhancement request

### DISCUSSION THREAD

**Discussion by SilvioGrosso** [rbreu/beeref#11] [Ideas & General Discussion]
Hello!

I am testing right now BeeRef on Windows 10.
It works great!
Thanks a lot for releasing this software :-)

One useful improvement, IMHO, would be the option to rotate the images 90° - 180° (anti and clockwise)
At present, it looks like you can only flip them vertically and horizontally.

EDIT:
I have just noticed you can rotate (any degree you wish) your images...
However, this option is not present in the menu "Transform" and I wrongly thought it was not implemented yet.

Btw, I don't know whether you know Krita...
This year they are also implementing their reference tool (Aka image docker) through a Google summer of code [1]

Best regards from Italy :-)

[1] https://docs.google.com/document/d/14M8rsLAfCT6QYblgK6Uz3Gh_n_fI7Idn1F3a_KbRb8c/edit

**Comment by rbreu** [comment]
Yes, Krita is my main art program. I'm curious to see how its reference tool is going to turn out. I use the current one now too, in combination with an external one. The more options the better.

It might be useful to have a couple of rotation options in the menu as well and thus accessible via keyboard shortcuts, but it's going to be a low priority for now.
